ci/start_init_test.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-set -x
+# set -x
```

Comment:
uncomment it.

Reply:
I will, after travis build passes successfully. Now it is difficult to understand the logs as it is too verbose